how-to-docs/amazon-resources.md

```diff
@@ -0,0 +1,55 @@
+# How to get Amazon Resources in the UK Cluster
+
+## Workflow
+1. Pull stacks-hod-platform repo
+2. Create/request credentials and secrets
+3. Configure templates in feature branch
+4. Create a pull request, and an _Amazon Resource Request_ in the hub
+5. After review, your pull request will be merged and your resources created
+
+### Pull the stacks-hod-platform repo
+```
+git clone ssh://git@gitlab.digital.homeoffice.gov.uk:2222/Devops/stacks-hod-platform.git
+cd stacks-hod-platformhttps://gitlab.digital.homeoffice.gov.uk/Devops/stacks-hod-platform/
```

Comment:
Pasted the link here by mistake, also update the repo name (confirm with Rohith what the prod repo will be called)